src/sage/schemes/hyperelliptic_curves/monsky_washnitzer.py

```diff
@@ -2385,12 +2396,12 @@ def __init__(self, Q, R=None, invert_y=True):
         if R is None:
             R = Q.base_ring()
 
-        # Github issue #9138: CommutativeAlgebra.__init__ must not be
+        # Github issue #9138: .__init__ must not be
         # done so early.  It tries to register a coercion, but that
         # requires the hash being available.  But the hash, in its
         # default implementation, relies on the string representation,
         # which is not available at this point.
-        # CommutativeAlgebra.__init__(self, R)  # moved to below.
+        # .__init__(self, R)  # moved to below.
```

Comment:
This comment does not apply because of the `UniqueRepresentation` (the hash is the id); so we can remove it.